src/cargo/core/compiler/custom_build.rs

```diff
@@ -725,13 +725,26 @@ impl BuildOutput {
         fn check_minimum_supported_rust_version_for_new_syntax(
             pkg_descr: &str,
             msrv: &Option<RustVersion>,
+            key: &str,
         ) -> CargoResult<()> {
             if let Some(msrv) = msrv {
                 let new_syntax_added_in = RustVersion::from_str("1.77.0")?;
                 if !new_syntax_added_in.is_compatible_with(msrv.as_partial()) {
+                    let prefix = format!("{key}=");
```

Comment:
Shouldn't this be inside of the `then` or inlined directly into that `format`?

Reply:
It was used one more place, but it's refactored now anyway.